src/Message/Timestamp.php

```diff
@@ -15,8 +15,8 @@ public function jsonSerialize(): string
 
     public function __toString(): string
     {
-        $me = clone $this;
+        $utc = $this->setTimezone(new \DateTimeZone('UTC'));
```

Comment:
`DateTimeImmutable#setTimezone` does not change the object, no need to `clone $this`

Reply:
Perfect, didn't think about this